main/lsp/requests/hover.cc

```diff
@@ -42,7 +42,7 @@ unique_ptr<ResponseMessage> HoverTask::runRequest(LSPTypecheckerDelegate &typech
 
     const core::GlobalState &gs = typechecker.state();
     auto result = LSPQuery::byLoc(config, typechecker, params->textDocument->uri, *params->position,
-                                  LSPMethod::TextDocumentHover);
+                                  LSPMethod::TextDocumentHover, false);
```

Comment:
false is `errorIfFileIsUntyped` which controls whether `byLoc` will return results or intentionally suppress them.

Reply:
at this point we may want to get rid of that, because i think that the vast majority and maybe all calls pass `false` at this point.